docs/release-notes/self-hosted/self-hosted-v3.2.0.md

```diff
@@ -0,0 +1,60 @@
+# Self-hosted v3.2.0
+
+These release notes are for [Codacy Self-hosted v3.2.0](https://github.com/codacy/chart/releases/tag/3.2.0){: target="_blank"}, released on December ##, 2020. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   Now, Codacy has an improved user experience to add and manage organizations and supports displaying an indefinite number of organizations per Git provider. (CY-1938)
```

Comment:
@inesbpinto @davidscordeiro, please check if you agree with the wording here. :+1: